### PR TITLE
Feature/notification documentation in api spec

### PR DESCRIPTION
**Pull request by stevenbal**
for https://github.com/VNG-Realisatie/gemma-zaken/issues/1231

### CODE REVIEW

File: src/ztc/api/kanalen.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from vng_api_common.notifications.kanalen import Kanaal
+
+from ztc.datamodel.models import BesluitType, InformatieObjectType, ZaakType
+
+KANAAL_BESLUITTYPEN = Kanaal(
+    'besluittypen',
+    main_resource=BesluitType,
+    kenmerken=()
```
Comment (sergei-maertens) [inline src/ztc/api/kanalen.py]: adding the catalogus name/identification would be a good kenmerk to filter on

File: src/ztc/api/kanalen.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+)
+
+KANAAL_INFORMATIEOBJECTTYPEN = Kanaal(
+    'informatieobjecttypen',
+    main_resource=InformatieObjectType,
+    kenmerken=()
```
Comment (sergei-maertens) [inline src/ztc/api/kanalen.py]: adding the catalogus name/identification would be a good kenmerk to filter on

File: src/ztc/api/kanalen.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+)
+
+KANAAL_ZAAKTYPEN = Kanaal(
+    'zaaktypen',
+    main_resource=ZaakType,
+    kenmerken=()
```
Comment (sergei-maertens) [inline src/ztc/api/kanalen.py]: adding the catalogus name/identification would be a good kenmerk to filter on

**Comment by joeribekker** [comment]
@stevenbal Please pick this up.

**Pull request merged**